src/pages/ChatMainPage/MobileVersion.js

```diff
@@ -162,7 +163,13 @@ const MobileVersion = () => {
             <div className={style.header_logo}>
               <img src={Logo} />
             </div>
-            <div className={style.header_search}>
+            <div
+              className={style.header_search}
+              onClick={() => {
+                navigate("/chats/search");
+                dispatch(getGroups());
+              }}
```

Comment:
It's better to write a separate function or method and use it here in onClick. Especially if it's length is more than 1 row.